service/matching/tasklist_test.go

```diff
@@ -21,9 +21,12 @@
 package matching
 
 import (
+	"fmt"
 	"strconv"
 	"testing"
 
+	"github.com/uber/cadence/common/persistence"
+
 	"github.com/stretchr/testify/require"
 )
```

Comment:
The lint failure indicate that you need to add `"github.com/uber/cadence/common/persistence"` below "github.com/stretchr/testify/require"
)